docs/content/en/install/local/_index.md

```diff
@@ -64,9 +64,9 @@ The `make NUM_CLUSTERS=<number> create-kind-multicluster` command, which is show
 
 **Tip:** kind generates a `kubeconfig` with the IP address of the API server set to `localhost` because the cluster is intended for local development. We need a `kubeconfig` with the IP address set to the internal address of the API server. The `setup-kind-mulitcluster.sh` script takes care of this requirement for you.  
 
-#### create-clientconfig
+#### k8ssandra-client
 
-[create-clientconfig.sh](https://github.com/k8ssandra/k8ssandra-operator/blob/main/scripts/create-clientconfig.sh) is in the k8ssandra-operator repo. This script is used to configure access to remote clusters, as described in subsequent topics. 
+The k8ssandra-client can be used to register data planes to control planes for multi-cluster use cases. See [here]({{< relref "tasks/data-plane-registration" >}}) for more details.
```

Comment:
**Suggestion:** 
```suggestion
The k8ssandra-client is a kubectl plugin which can be used to register data planes to control planes for multi-cluster use cases. See [here]({{< relref "tasks/data-plane-registration" >}}) for more details.
```

Reply:
It can be used standalone too (which is probably what I'd do). I think referring to it as a kubectl plugin is extra detail that is more likely to confuse users than help them to be honest.